AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/MAX/Rewarded/ALMAXRewardedAdViewController.m

```diff
@@ -25,7 +25,7 @@ - (void)viewDidLoad
     self.rewardedAd = [MARewardedAd sharedWithAdUnitIdentifier: @"YOUR_AD_UNIT_ID"];
     
     self.rewardedAd.delegate = self;
-    self.rewardedAd.revenueDelegate = self;
+    self.rewardedAd.delegate = self;
```

Comment:
Kind of confused - why are we setting `delegate` twice? I think it should be `revenueDelegate`.